File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/commands/SeatAssignments/UnassignSeat.cs
using System;
using ENode.Commanding;

namespace Registration.Commands.SeatAssignments
{
    public class UnassignSeat : Command<Guid>
    {
        public int Position { get; set; }

        public UnassignSeat() { }
        public UnassignSeat(Guid orderId) : base(orderId) { }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/registration/commands/seatassignments/UnassignSeat.java
package com.microsoft.conference.common.registration.commands.seatassignments;

import org.enodeframework.commanding.Command;

public class UnassignSeat extends Command<String> {
    public int position;

    public UnassignSeat() {
    }

    public UnassignSeat(String orderId) {
        super(orderId);
    }
}
